### PR TITLE
[stable9] Fix incompatible syntax

### PR DESCRIPTION
**Pull request by VicDeo**
PHP 5.4 does not allow that
In stable9.1 already
Not needed for master

### CODE REVIEW

**Comment by PVince81** [comment]
@VicDeo please ping me on PRs where you want reviews...

File: src/Console/Application.php [hunk 1]
```diff
@@ -200,7 +200,7 @@ protected function doRunCommand(Command $command, InputInterface $input, OutputI
 			$command->setContainer($this->getContainer());
 			$commandName = $this->getCommandName($input);
 			$this->getLogger()->info('Execution of ' . $commandName . ' command started');
-			if (!empty($command->getMessage())){
+			if ($command->getMessage() == !''){
```
Comment (PVince81) [inline src/Console/Application.php]: equal equal not empty string ? Looks suspicious

**Comment by VicDeo** [comment]
@PVince81 not equal to empty string is better indeed.
Fixed.

File: src/Console/Application.php [hunk 2]
```diff
@@ -200,7 +200,7 @@ protected function doRunCommand(Command $command, InputInterface $input, OutputI
 			$command->setContainer($this->getContainer());
 			$commandName = $this->getCommandName($input);
 			$this->getLogger()->info('Execution of ' . $commandName . ' command started');
-			if (!empty($command->getMessage())){
+			if ($command->getMessage() ==! ''){
```
Comment (PVince81) [inline src/Console/Application.php]: shouldn't this be `!==` ?

**Comment by VicDeo** [comment]
@PVince81 oops. I need coffee...

**Comment by VicDeo** [comment]
@PVince81 is it ok now?

**Comment by PVince81** [comment]
@VicDeo yes, if you think that empty string is enough.
If the command message can also be null, then it might not be suitable.

:+1: please merge if empty string is enough

**Comment by VicDeo** [comment]
@PVince81 yes, it's enough https://github.com/owncloud/updater/blob/master/src/Command/Command.php#L35-L45

**Pull request merged**